terraform/_gen/main.go

```diff
@@ -40,6 +40,10 @@ type payload struct {
 	Kind string
 	// DefaultVersion represents the default resource version on create
 	DefaultVersion string
+	// StaticID states whether this particular resource has a static (usually 0) Metadata.ID
+	// This is relevant to cache enabled clusters: we use Metadata.ID to check if the resource was updated
+	// Currently, the resources that don't have a dynamic Metadata.ID are strong consistent: oidc, github and saml connectors
+	StaticID bool
```

Comment:
Can this be `HasStaticID` - I'd read `StaticID` as the static ID of something, rather than a predicate.

Reply:
Yes, sounds better
Thank you